serverless.yml

```diff
@@ -0,0 +1,114 @@
+# Welcome to Serverless!
+#
+# This file is the main config file for your service.
+# It's very minimal at this point and uses default values.
+# You can always add more config options for more control.
+# We've included some commented out config examples here.
+# Just uncomment any of them to get that config option.
+#
+# For full config options, check the docs:
+#    docs.serverless.com
+#
+# Happy Coding!
+
+service: st-joseph-email
+# app and org for use with dashboard.serverless.com
+#app: your-app-name
+#org: your-org-name
+
+# You can pin your service to only deploy with a specific Serverless version
+# Check out our docs for more details
+frameworkVersion: '2'
+
+provider:
+  name: aws
+  runtime: nodejs12.x
```

Comment:
Local runtime in developement for us is 14.x we have to make sure that is okay to develop in 14.x and deploy in 12.x. Should be fine, it worked fine for JAMFREE

Reply:
https://github.com/serverless/serverless/blob/master/CHANGELOG.md